plugin.xml

```diff
@@ -15,33 +18,25 @@
         <config-file target="AndroidManifest.xml" parent="/manifest/application">
             <service android:name="io.radar.cordova.RadarForegroundService" android:foregroundServiceType="location" android:stopWithTask="true" />
         </config-file>
-        <config-file target="AndroidManifest.xml" parent="/manifest/application">
-            <receiver
-                    android:name="io.radar.cordova.RadarCordovaPlugin$RadarCordovaReceiver"
-                    android:enabled="true"
-                    android:exported="false">
-                <intent-filter>
-                    <action android:name="io.radar.sdk.RECEIVED" />
-                </intent-filter>
-            </receiver>
-        </config-file>
 
-        <edit-config file="AndroidManifest.xml" target="/manifest" mode="merge">
-            <manifest xmlns:tools="http://schemas.android.com/tools" />
-        </edit-config>
+        <source-file src="src/android/radar.xml" target-dir="res/values" />
+
+        <config-file target="./res/values/radar.xml" parent="/resources">
```

Comment:
During my initial research, I found that it's recommended to avoid touching the strings.xml file because this file is important and to avoid conflicts with other plugins. [Other plugins](https://github.com/cordova-plugin-facebook-connect/cordova-plugin-facebook-connect/blob/master/src/android/facebookconnect.xml) follow this approach